index.bs

```diff
@@ -1791,11 +1791,17 @@ The following extended attributes are applicable to constants:
 
 <pre class="grammar" id="prod-ConstValue">
     ConstValue :
+        UndefinedLiteral
```

Comment:
Is this for #905?

Reply:
No, unrelated. It just looked correct, since I was adding a new literal, that it should be added to the Const productions as well.

Reply:
Does this PR uses the new literal elsewhere? I think this is the only use.

Reply:
> Does this PR uses the new literal elsewhere? I think this is the only use.

No, this is it. But the other primitives all have a literal, except null (which is a little weird, but it's handled as a type annotation rather than a type on its own, so eh).

I could remove this if necessary, it just seemed nice.

Reply:
It'd probably be better to avoid unnecessary indirections in the grammar.